Update radio.md - say to use role="radiogroup" not role="group"

bawolff opened the pull request:
The text says role="group" but the example does role="radiogroup". Looking at the aria docs, it sounds like radiogroup is more correct.

egardner commented:
Here's a question – if I'm in the `vue-components` package and I want to use some of these tokens during component development, is the assumption that I will always need to run the build script for the tokens first? Just trying to figure out the proper workflow here.

egardner commented:
Also @tzhelyazkova looks like this issue is related to the CLI type-error (which I am seeing locally as well): https://github.com/amzn/style-dictionary/issues/680.

egardner commented on packages/design-tokens/package.json:
```diff
@@ -1,3 +1,22 @@
+	},
+	"scripts": {
+		"build": "yarn run clean && yarn run build:tokens",
+		"build:tokens": "style-dictionary build -c ./.style-dictionary/config.js",
```
This was the source of the `TypeError`. Looks like if we configure StyleDictionary using the JS API we need to just execute that file in Node directly, rather than using the `style-dictionary` binary. Changing the line to this fixes the error:

```		
"build:tokens": "node ./.style-dictionary/config.js",
```

I can update this momentarily.

tzhelyazkova commented:
> Here's a question – if I'm in the `vue-components` package and I want to use some of these tokens during component development, is the assumption that I will always need to run the build script for the tokens first? Just trying to figure out the proper workflow here.

Yes, you'll have to run the build command first.
I'll look into adding a `watch` feature in a separate PR, which will allow us to have any changes from the tokens in the vue-components without having to build every time.

bawolff committed:
Update radio.md - say to use role="radiogroup" not role="group"

The text says role="group" but the example does role="radiogroup". Looking at the aria docs, it sounds like radiogroup is more correct.
